packages/yum/install_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -exu
+
+package="$1"
+
+os_version=$(cut -d: -f5 /etc/system-release-cpe)
+mariadb_version=$(echo "${package}" | cut -d'-' -f2)
+
+cat > /etc/yum.repos.d/MariaDB.repo <<EOF
```

Comment:
rootじゃなくても動くようにするならこうだよ。

```suggestion
cat <<EOF | sudo tee /etc/yum.repos.d/MariaDB.repo
```

Reply:
おぉ。なるほど。ありがとうございます。